app/components/enterprise-banner.js

```diff
@@ -101,9 +101,22 @@ export default Component.extend({
     return (isPaid && check);
   },
 
-  @computed('isPaid', 'almostExceedingSeats', 'exceedingSeats')
-  showSeatsBanner(isPaid, almostExceeding, exceeding) {
-    return (isPaid && (almostExceeding || exceeding));
+@computed('isPaid', 'checkSeatsBanner')
```

Comment:
I think the indenting is messed up here? Same below.

Reply:
it looks weird in the browser but is fine in my text editor. not sure what the issue is...